pixeltable/exprs/function_call.py

```diff
@@ -146,6 +146,7 @@ def __init__(
         self.id = self._create_id()
 
     def _create_rowid_refs(self, tbl: catalog.Table) -> list[Expr]:
+        assert isinstance(tbl, catalog.table.LocalTable)
```

Comment:
Why?

Reply:
Because it needs to access _tbl_version_path